SDC_DB/createSchema.js

```diff
@@ -54,8 +72,8 @@ function write10Million(start = 1e2) {
   }
 
   if (i === 0) {
-    const command = 'mongoimport -d silverspoon -c restaurantMenus --file db/testData.json --numInsertionWorkers 4';
-    exec(command);
+    const command = 'mongoimport -d silverspoon -c restaurantmenus --file db/testData.json --type json --numInsertionWorkers 4';
```

Comment:
just curious if the flag --type json was required for the command to seed properly?